packages/grpc-js/src/server-call.ts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {EventEmitter} from 'events';
+import {Duplex, Readable, Writable} from 'stream';
+import {ServiceError} from './call';
+import {Deserialize, Serialize} from './make-client';
+import {Metadata} from './metadata';
+
+
+export class ServerUnaryCall<RequestType> extends EventEmitter {
+  cancelled: boolean;
+  metadata: Metadata;
+  request: RequestType|null;
+  private call: ServerCall;
+
+  constructor(call: ServerCall, metadata: Metadata) {
```

Comment:
For this constructor and others, arguments that are assigned directly to instance members should be declared and assigned directly in the constructor, like `constructor(private call: ServerCall, public metadata: Metadata)`, then you can delete both the declaration lines and the assignment lines for those members.

Reply:
I've updated the PR to addressed this. I'm not sure if there is a tslint rule for this or not. If so, it might be worth enabling.